### PR TITLE
Add basic Pest architecture tests

### PR DESCRIPTION
**Pull request by ash-jc-allen**
Hey @jbrooksuk! This PR adds some new Pest architecture tests that I've quickly put together.

I wasn't sure how in-depth and strict you'd want them to be, so I've kept them all fairly lax. But I'm hoping they'll add some value by enforcing some conventions for future contributions.

If there's anything you'd like me to change, please give me a shout 😄

### CODE REVIEW

File: tests/Architecture/ActionsTest.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+test('actions test')
+    ->expect('Cachet\Actions')
+    ->toBeClasses()
+    ->toExtendNothing()
+    ->toHaveMethod('handle');
```
Comment (jbrooksuk) [inline tests/Architecture/ActionsTest.php:7]: I think we should also check that Actions use the `AsAction` trait for consistency.

File: tests/Architecture/ViewTest.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Contracts\Validation\InvokableRule;
+
+test('view test')
+    ->expect('Cachet\View')
+    ->toBeClasses()
+    ->toExtend(\Illuminate\View\Component::class);
```
Comment (jbrooksuk) [inline tests/Architecture/ViewTest.php:8]: We can import this class in full.

**Review by jbrooksuk — CHANGES_REQUESTED** [review]
Thanks @ash-jc-allen! Just two comments.

File: tests/Architecture/ViewTest.php (hunk 2 above)
Comment (ash-jc-allen) [reply tests/Architecture/ViewTest.php]: Ooops! I'm not too sure why I didn't import that sorry 🤦‍♂️ That should be fixed now!

**Comment by jbrooksuk** [comment]
@ash-jc-allen tests still fail unfortunately.

**Comment by ash-jc-allen** [comment]
Sorry @jbrooksuk, I got distracted and completely forgot to fix them haha! I've just had a look at them and I think they're failing for the `prefers-lowest` because it's using `pestphp/pest` v2.13.0, but the architecture `toHaveMethod` expectation was only added last week in v2.17.0.

It looks like there was already a `toHaveMethod` expectation earlier in Pest, but it worked slightly different to the newer version.

For reference: https://github.com/pestphp/pest/issues/951

So I've just removed that expectation for the time being. Hopefully this is all okay? 🙂

**Review by jbrooksuk — APPROVED** [review]

**Pull request merged**